server/plugin/command.go

```diff
@@ -43,6 +43,20 @@ const (
 	deleteAll = "delete-all"
 )
 
+type Features string
+
+func (features Features) String() string {
+	return string(features)
+}
+
+func (features Features) ToString() string {
+	return "`" + strings.Join(strings.Split(string(features), ","), "`, `") + "`"
+}
```

Comment:
Why are there two methods? Should `Features` always be printed the same way?

Reply:
@hanzei wanted to highlight the features so used it , i prefer it to be highlighted 
let me know your view view on this

Reply:
Using a code block looks good to me. But why is there also a `String()` method?